rendercv/data/models/curriculum_vitae.py

```diff
@@ -414,6 +418,13 @@ class CurriculumVitae(RenderCVBaseModelWithExtraKeys):
         alias="sections",
     )
 
+    @pydantic.field_validator("photo")
+    @classmethod
+    def photo_path(cls, value: str | Path, info: pydantic.ValidationInfo) -> Path:
+        """Cast `photo` to Path and make the path absolute"""
+        path = Path(value)
+        return path.absolute()
+
```

Comment:
With this snippet I compute the absolute path of the image. This resolve the problem that relatives path references should change in the j2 template compilation context.

However there is still a problem: relative path is always relative to the cwd, this is the expected behaviour when the parameter is passed by args (ex: --cv.photo=propic.jpg) but I don't think is the correct one if the path is specified in the yaml, because the path should be relative to the yaml and not to the cwd. I think this problem cannot be solved in this PR, as the code merge together YAML and args data before passing it to the pydantic model.

Reply:
The line below
https://github.com/sinaatalay/rendercv/blob/9cbff56057ac552dc15ec0e1160c2a740921096a/rendercv/cli/commands.py#L212

changes the current working directory to the parent of the input file. If we move that line above the line below

https://github.com/sinaatalay/rendercv/blob/9cbff56057ac552dc15ec0e1160c2a740921096a/rendercv/cli/commands.py#L196

then, the problem will be resolved and the path will always be relative to the input file path's directory.

Reply:
Using os.chdir to change the relative point solve the problem of the relative reference from the YAML but then you have that from the CLI --cv.photo would still be relative to the YAML file and not to the cwd (which I think it might be misleading).

Reply:
The only quick solution I see is to explicit the arg "--cv.photo" and, if passed, compute the absolute path from the cwd (idk how typer works, I generally use click, tomorrow I'll take a look to the docs).

Reply:
I see your point, but I think it's still fine because we clearly define what that option does, which is overriding the YAML file. So users should expect that `--cv.photo PATH` is equivalent to writing
```yaml
cv:
  photo: PATH
```
which will be equivalent.

Reply:
Makes sense, during next days I'll proceed as you suggested. Thank you for the feedback.